content/kubermatic/master/architecture/concepts/automation_accounts/_index.en.md

```diff
@@ -0,0 +1,104 @@
++++
+title = "Automation Accounts"
+date = 2020-03-31T12:01:35+02:00
+weight = 15
+
++++
+
+## Introduction
+
+An automation account is a special type of account intended to represent a non-human user that needs to authenticate and be
+authorized to access resources in Kubermatic APIs.
+
+Automation accounts are used in scenarios such as:
+
+- Running workloads on user clusters.
+- Running workloads which are not tied to the lifecycle of a human user.
+
```

Comment:
- Creating fully functional GitOps workflows that automate everything end to end